tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -48,6 +48,8 @@ def tearDown(self):
         session.query(DagRun).delete()
         session.commit()
         session.close()
+        from airflow.configuration import conf as global_conf
+        global_conf.set('core', 'store_serialized_dags', 'False')
```

Comment:
Can you use conf_vars to avoid side-effect?

Reply:
I think it's worth testing two cases - when store_serialized_dags = true and when store_serialized_dags = False

Reply:
Done